tests/queries/test_q.py

```diff
@@ -3,6 +3,30 @@
 
 
 class QTests(SimpleTestCase):
+    def test_and_falsey(self):
```

Comment:
Hmm, I feel that it shouldn't pass. An unbalanced OR with one side being `x` shouldn't equal just `x`. IMO the first case shouldn't have been possible in the first place, but it is an independent and different thing than just `x`.